File: trail1/굉장한 수/amazing-number.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int N;
    cin >> N;
    string res = (N%2==1 && N%3==0) || (N%2==0 && N%5==0) ? "true" : "false";
    cout << res << " ";
    return 0;
}

File: trail1/굉장한 수/amazing-number.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        String res = (N%2==1 && N%3==0) || (N%10==0) ? "true" : "false";
        System.out.println(res);
    }
}